app/views/categories/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>category</title>
+</head>
+<body>
+	<br>
+	<br>
+	<%= form_with model: @category do |form| %>
+	<%= form.label :title, "title : " %>
+	<%= form.text_field :title %>
+	<%= form.submit "create" %>
```

Comment:
Create

Reply:
create -> Create and also handled intendation for space